documentation/commands/git-sync.md

```diff
@@ -15,9 +15,9 @@ git sync (--abort | --continue | --skip)
 Synchronizes the current branch with the rest of the world.
 
 When run on a feature branch
-* syncs the main branch
+* syncs all ancestor branches including the main branch
```

Comment:
do we need to mention `including the main branch`? Since the main branch is an ancestor of all feature branches I think thats implied